interfaces/Config/templates/config_general.tmpl

```diff
@@ -372,6 +402,40 @@
         }
     })
 
+    // Upload config
+    \$('.uploadConfig').click(function(event){
+        removeObfuscation()
+        var theButton = \$(this)
+        var resultBox = theButton.parents('.col1').find('.result-box .alert');
+        theButton.attr("disabled", "disabled")
+        theButton.find('span').toggleClass('glyphicon-sort glyphicon-refresh spin-glyphicon')
+        formData = new FormData(document.querySelector('#uploadConfigForm'))
+        \$.ajax({
```

Comment:
Is this really needed? Why not use the regular `.fullform` triggers? Seems like a lot of code-duplication.
https://github.com/sabnzbd/sabnzbd/blob/develop/interfaces/Config/templates/staticcfg/js/script.js#L344

Reply:
I hadn't noticed that script. I've never been able to follow JS and CSS. I started with the script for testing a server and changed it using cut and paste and a lot of fiddling to make it work with file upload.

I've changed it to use the existing system. However, now it won't be imported immediately if the user answers no to the restart request. It will be imported later if there is a restart but not if it's shut down. That might be unexpected to the user.